pkg/middlewares/tracing/entrypoint_test.go

```diff
@@ -7,87 +7,67 @@ import (
 	"testing"
 
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 	"github.com/traefik/traefik/v3/pkg/tracing"
 	"go.opentelemetry.io/otel/trace"
 )
 
 func TestEntryPointMiddleware(t *testing.T) {
 	type expected struct {
-		Tags          map[string]interface{}
-		OperationName string
+		Tags     map[string]interface{}
+		SpanName string
 	}
 
 	testCases := []struct {
-		desc          string
-		entryPoint    string
-		spanNameLimit int
-		tracing       *trackingBackenMock
-		expected      expected
+		desc       string
+		entryPoint string
+		tracing    tracing.Tracer
+		expected   expected
 	}{
 		{
-			desc:          "no truncation test",
-			entryPoint:    "test",
-			spanNameLimit: 0,
-			tracing: &trackingBackenMock{
-				tracer: &MockTracer{Span: &MockSpan{Tags: make(map[string]interface{})}},
-			},
-			expected: expected{
-				Tags: map[string]interface{}{
-					"span.kind":                   trace.SpanKindServer.String(),
-					"http.method":                 http.MethodGet,
-					"component":                   "",
-					"http.url":                    "http://www.test.com",
-					"http.host":                   "www.test.com",
-					"http.client_ip":              "10.0.0.1",
-					"http.request_content_length": int64(0),
-					"http.user_agent":             "entrypoint-test",
-				},
-				OperationName: "EntryPoint test www.test.com",
-			},
-		},
-		{
-			desc:          "basic test",
-			entryPoint:    "test",
-			spanNameLimit: 25,
-			tracing: &trackingBackenMock{
-				tracer: &MockTracer{Span: &MockSpan{Tags: make(map[string]interface{})}},
-			},
+			desc:       "basic test",
+			entryPoint: "test",
+			tracing:    &MockTracer{Span: &MockSpan{Tags: make(map[string]interface{})}},
 			expected: expected{
 				Tags: map[string]interface{}{
-					"span.kind":                   trace.SpanKindServer.String(),
-					"http.method":                 http.MethodGet,
-					"component":                   "",
-					"http.url":                    "http://www.test.com",
-					"http.host":                   "www.test.com",
-					"http.client_ip":              "10.0.0.1",
-					"http.request_content_length": int64(0),
-					"http.user_agent":             "entrypoint-test",
+					"span.kind":                 trace.SpanKindServer.String(),
```

Comment:
worth of double checking all of these variable names before merge. Seems like this world is changing every day currently.